database/DatabaseReference.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,7 +2,7 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:DatabaseReference.xcodeproj">
```

Comment:
This should definitely not be committed

Reply:
I had no idea where it came from, so assumed it was an Xcode upgrade issue. Will revert.

Reply:
This should be fixed now.